lang/cs/Org.Apache.REEF.Evaluator/Evaluator.cs

```diff
@@ -249,7 +202,8 @@ private static RuntimeClock InstantiateClock()
             IConfiguration clockConfiguration = new ConfigurationModuleBuilder().Build().Build();
```

Comment:
Shouldn't the `Clock` be injected using the configuration provided for the Evaluator?